README.md

```diff
@@ -74,7 +76,33 @@ on:
       - '**.webp'
 ```
 
-The above workflow will only run when `jpg`, `png` or `webp` files are changed.
+The above workflow will only run on a pull request when `jpg`, `png` or `webp` files are changed.
+
+## Migration Steps

```

Comment:
This section requires more explanation. What is the migration? From what? What are the major differences? 
Perhaps this section would be better suited as a changelog?

Reply:
The migration is basically the changes that have made to this action since it was released. Those were the differences from my configuration I noticed after looking at the warning from the docker container. I'm not sure when they changed but a quick history check would show that. 

If you commit to maintain a change log with migration steps that would be great. You could link to it in the container that is showing the warning. But for the time being, I think this is better than what was there before to satisfy the warning from the container.